src/styles/blocks/catalog.scss

```diff
@@ -21,5 +22,14 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition-property: transform;
+    transition-duration: 300ms;
+    transition-timing-function: linear;
+  }
+
+  &__card:hover {
+    color: #34568b;
```

Comment:
You also changing color< but trasition is only for trasform. Also you can use short hand
`transition: color 0.3s, transform 0.3s linear`